src/index.html

```diff
@@ -1,13 +1,52 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="product-container" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="iMAC 27"
+        class="product-container__img"
+      >
+      <div class="product-container__description">
+        <h1 class="product-container__card-header">
+          Монoблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </h1>
+        <div class="product-container__code">Код товара: 195434</div>
+        <div class="rating">
+          <div class="rating__container">
+            <div class="star"></div>
```

Comment:
Let's use BEM here as well. No need to create a new BEM-block for each of the stars. Let them be elements of